src/Microsoft.VisualStudio.Services.Agent/ProcessWrapper.cs

```diff
@@ -0,0 +1,295 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.Services.Agent
+{
+    /**/
+    public class ProcessWrapper
```

Comment:
Can we merge this class into ProcessInvoker so there is only one wrapper over a process?